src/include/connection.h

```diff
@@ -11,7 +11,7 @@
  *******************************************/
 /*
  * WT_PROCESS --
- *	Per-process information for the library.
+ *  Per-process information for the library.
```

Comment:
In this particular instance, the formatting in develop is weird: instead of spaces it uses a two-space tab. Do we want to require everyone to use this style in their comments given that we all use different editors? I understand asking to replace four spaces with a tab, but develop doesn't do that (and I fixed my editor to not to that), but replacing two spaces with a two-space tab (which is what develop does, according to my emacs), is a bit obscure. Let's hope that the new version doesn't flag too many of these.

Reply:
I'll mark this as resolved for now, we will resurrect it if it's still present in the new commit.